src/sections/FAQs.tsx

```diff
@@ -1,5 +1,101 @@
+    },
+    {
+      question: 'How can I do it anonymously?',
+      answer: 'ZK Email leverages the principles of Zero Knowledge proofs and serverless execution within the browser to provide a verifiable yet anonymous way to confirm an emails contents and recipents . Zero Knowledge proofs allow provers to prove that they know or possess certain information without revealing the information itself to a verifer. In ZK Email, this technology is used to verify user identity and email content without exposing sensitive data to a server or other users. The serverless architecture means that all proof generation is executed entirely within the user`s browser.'
+    },
+    {
+      question: 'Why don’t I need to trust you?',
+      answer: 'The DKIM email signitures do not contain information we can use to sign other emails, all our code is open source and being audited'
+    },
+    {
+      question: 'How do you verify the email contents?',
+      answer: 'We use the existing DKIM Signiture, almost all emails are signed by the sending domain server using an algorithm called DKIM. It can be summarized as this rsa_sign(sha256(from:..., to:..., subject:..., <body hash>,...), private key). Every time an email is sent we can verify the sender, receiver, the subject, the body by checking if the corresponding public key of the email address applied on the DKIM signiture returns the sha256(from:..., to:..., subject:..., <body hash>,...) a hash of the from, to subject, email details which we can check by rehashing'
+    }
+  ];
+
+  return (
+    <div className="w-full mx-auto mt-10 z-10 px-8 sm:px-36">
+      <div className='flex flex-row justify-between'>
+        <div>
+          <h1 className="text-3xl font-semibold mb-6 h3 text-left " style={{textAlign:'left'}}>FAQs</h1>
+          <p className="text-left mb-4 text-[#A8A8A8]">
+            Quick answers to the common questions you might have. <br className='hidden md:inline'></br>
+            Can’t find what you looking for? Read our docs or <a href="/contact" className=" underline text-[#F5F3EF]">contact us</a>
+          </p>
+        </div>
+        <div>
+          <a  href='https://zkemail.gitbook.io/zk-email' target='_blank'>
+            <button className='flex flex-row justify-between gap-1 items-center px-[12px] py-[8px] bg-[#1C1C1C] border-solid border-2 border-[#3B3B3B] rounded-[12px] text-[#F5F3EF] text-[16px] whitespace-nowrap'>
+              Read Docs
+              <svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 14 14" fill="none">
```

Comment:
It seems like you are using the same svg file many times but pasted the whole content everywhere (like this right arrow sign svg). It would be better if you can define it in one file and use it everywhere else